config/waagent.conf

```diff
@@ -9,6 +9,9 @@ Provisioning.Enabled=y
 # backup, monitoring, or any extension handling whatsoever.
 Extensions.Enabled=y
 
+#Enable extensions FastTrack, which improves goal state processing times
```

Comment:
Let's document the setting in README.md

Reply:
We talked offline about not adding the flag explicitly in the conf file. 

Actually, to make it easier for people to change it, it may be better to leave it, probaly adding to the comment something like "Do not disable this setting unless directed by Microsoft Support"?

Reply:
I dont feel we should add this flag at all here, as this feature doesnt impact customers directly at all, its just modifying the internal functioning of the whole goalstate model. We shouldn't give this power to the customer